main.go

```diff
@@ -12,6 +12,7 @@ import (
 
 func main() {
 	numericParameter := flag.Bool("n", false, "Sort numbers from lowest to highest")
+	writingParameter := flag.String("o","","Save result into a file")
```

Comment:
This formatting differs a bit from the other one.

Reply:
i dont exactly get your point

Reply:
You forgot the code formatting.